src/mmw/js/src/analyze/views.js

```diff
@@ -481,6 +476,32 @@ var AnalyzeLayerToggleDropdownView = Marionette.ItemView.extend({
     }
 });
 
+var AnalyzeDescriptionView = Marionette.LayoutView.extend({
+    template: AnalyzeDescriptionTmpl,
+    regions: {
+        layerToggleRegion: '.layer-region',
+    },
+
+    onShow: function() {
+        var associatedLayerCodes = this.model.get('associatedLayerCodes');
+        if (associatedLayerCodes) {
+            if (associatedLayerCodes.length === 1) {
+                this.layerToggleRegion.show(new AnalyzeLayerToggleView({
+                    code: _.first(associatedLayerCodes)
+                }));
+            } else {
+                this.layerToggleRegion.show(new AnalyzeLayerToggleDropdownView({
+                    collection: new Backbone.Collection(
+                        _.map(associatedLayerCodes, function(code) {
+                            return App.getLayerTabCollection()
+                                .findLayerWhere({ code: code });
```

Comment:
This is a little difficult to read. Can we make it so that this aligns with the `.` on the line above?

```js
return App.getLayerTabCollection()
          .findLayerWhere({ code: code });
```